examples/worlds/actors_population.sdf.erb

```diff
@@ -138,13 +138,14 @@
     </include>
 
     <%
-      # number of population
-      total = 10
-      for i in (0..total-1)
-        for j in (1..total)
+      # population = column * row
+      column = 100
+      for j in (0..column-1)
+        row = 100
```

Comment:
How about making the number of rows and columns a command line option which defaults to 10? This should give users more flexibility and we don't need to commit all the generated combinations.

Here's a snippet that we've used in the past to parse command line options of the format `[opt]=[arg]`

```
  vars = ARGV.take_while {|arg| arg[/^\w+=/]}
  ARGV.slice!(0, vars.size)
  vars.each do |var|
    k, v = var.split('=', 2)
    TOPLEVEL_BINDING.eval %Q(#{k} = "#{v}")
  end
```

Then you can expose options like `e` and `c`:

`erb r=10 c=5 actors_population.sdf.erb > actors_population.sdf`

And inside the code, use:

```
      column = c.to_i
      row = r.to_i
```

Reply:
I pushed the cli parsing code, but am not sure how to set default command line option to 10. Any pointer's appreciated!

Reply:
How about 

```
column = 10
if (defined? c)
  column = c.to_i
end
```